Mathlib/Computability/Encoding.lean

```diff
@@ -111,23 +112,23 @@ theorem encodePosNum_nonempty (n : PosNum) : encodePosNum n ≠ [] :=
   PosNum.casesOn n (List.cons_ne_nil _ _) (fun _m => List.cons_ne_nil _ _) fun _m =>
     List.cons_ne_nil _ _
 
-theorem decode_encodePosNum : ∀ n, decodePosNum (encodePosNum n) = n := by
+@[simp] theorem decode_encodePosNum : ∀ n, decodePosNum (encodePosNum n) = n := by
```

Comment:
Standard question if the \forall should be left of the colon. (But this is pre-existing.)

Reply:
So not relevant here. :-)